.circleci/config.yml

```diff
@@ -183,14 +205,24 @@ workflows:
             branches:
               only:
                 - master
+      - rahti-demo:
+          requires:
+            - build
+            - war
+          filters:
+            tags:
+              only: /v[0-9]\.[0-9].*/
+            branches:
+              only:
+                - master
```

Comment:
this means it will also deploy every version of master. I think you want `only: []` to disable deploying any branches to rahti-demo.